cocos/tween/actions/action.ts

```diff
@@ -279,4 +279,6 @@ export abstract class FiniteTimeAction extends Action {
     abstract clone (): FiniteTimeAction;
 
     abstract reverse (): FiniteTimeAction;
+
+    abstract isUnknownTime (): boolean;
```

Comment:
It is better to rename to `isUnknownDuration`

Reply:
Or rename to `isDynamicDuration`

Reply:
Yep, it's a better name. I will update it.

Reply:
Updated